prismic-model/.gitignore

```diff
@@ -1,2 +1,3 @@
 snapshot.*
 node_modules
+contentReport.json
```

Comment:
I thought we had a lint rule that would add an empty line of whitespace at the end of a file (if it wasn't present)?